Tests/SwiftDocCTests/Utility/ListItemExtractorTests.swift

```diff
@@ -15,6 +15,65 @@ import Markdown
 
 class ListItemExtractorTests: XCTestCase {
     
+    func testSupportsSpacesInTaggedElementNames() throws {
+        let testSource = URL(fileURLWithPath: "/path/to/test-source-\(ProcessInfo.processInfo.globallyUniqueString)")
+        func extractedTags(_ markup: String) -> TaggedListItemExtractor {
+            let document = Document(parsing: markup, source: testSource, options: .parseSymbolLinks)
+            
+            var extractor = TaggedListItemExtractor()
+            _ = extractor.visit(document)
+            return extractor
+        }
+    
+        for whitespace in [" ", "   ", "\t"] {
+            let parameters = extractedTags("""
+            - Parameter\(whitespace)some parameter with spaces: Some description of this parameter.
```

Comment:
Nit: It looks like we're missing some tests for when there's nothing after the list item tag (e.g. just `- Parameter: Some description of this parameter.`, would it be worth adding some?

Reply:
Although now, that I read those tests I see that I'm verifying the wrong lists for the last two tests.

Reply:
> Although now, that I read those tests I see that I'm verifying the wrong lists for the last two tests.

Fixed in https://github.com/swiftlang/swift-docc/pull/1017/commits/093ed59cb6deedf09b35787008bde975ad2136e9